### PR TITLE
Update Vue (now uses vue cli)

### PR DESCRIPTION
**Pull request by Serubin**
Update Vuejs to latest version. Newer versions of Vue provide an improved build system and better upgrading and integration of tools.

This provides a nice launching point to cleaning up the code base.

See #82 for more details

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,73 +26,40 @@
     "linkifyjs": "^2.1.8",
     "object-hash": "^1.3.1",
     "reconnecting-websocket": "^3.2.2",
-    "vue": "~2.5.22",
+    "vue": "^2.6.10",
     "vue-flatpickr-component": "^7.0.6",
     "vue-h-ajax": "^0.1.3",
     "vue-i18n": "^8.11.1",
-    "vue-router": "^3.0.6",
+    "vue-router": "^3.0.3",
```
Comment (klinker24) [inline package.json]: Downgraded the router?

**Comment by klinker24** [comment]
All seems to work well, for me. There is that error you have in #85, but it doesn't seem to affect anything at this time. Going to test this throughout today, then I will get it merged in 👍

File: package.json (hunk 1 above)
Comment (Serubin) [reply package.json]: Vue-cli has a mind of it's own. I'll manually upgrade it.

**Comment by Serubin** [comment]
`feature/improve-worker` and `fix-linting` branches both rely on this to merge first. Just an fyi.

**Pull request merged**